packages/backend-data/src/types.ts

```diff
@@ -99,6 +102,14 @@ export type AuthorizationModes = {
   oidcAuthorizationMode?: OIDCAuthorizationModeProps;
 };
 
+/**
+ * Schemas type definition, can be either a raw Graphql string, or a typed model schema, or a collection of combined Schemas.
+ */
+export type DataSchemasCollection =
```

Comment:
This probably isn't the right name.  This can now be:
- A string schema representation
- A single object schema representation
- An object containing many schemas

Reply:
I think it's fine to have a type like `DataSchema = string | DerivedModelSchema | DerivedCombinedSchema`

Reply:
Good point. I need both to represent the un-normalized input schema vs the DataSchema that is being processed (never a combined schema), I'm changing this to `DataSchemaInput` and `DataSchema` to better represent this distinction.